tests/docker/docker-compose.yml

```diff
@@ -30,6 +30,12 @@ services:
     - redpanda-test
   rp:
     image: vectorized/redpanda-test-node
+    deploy:
+      resources:
+        limits:
+          cpus: '0.75'
+        reservations:
+          cpus: '1'
```

Comment:
I don't really understand this change. Presumably something else (e.g. CI runner or locally `task`) is launching a set of containers. How does constraining the CPU reservations here help if we start an arbitrary set of containers to use with ducktape? Generally we start redpanda in containers with more than 1 CPU so that we always have parallelism inside of redpanda. Will this cause it to be slower? What about locally? How will I be constrained in the local dev ducktape runs?

Reply:
more than constraining, the goal is to see if pinning each container to a specific number of CPUs would help with resource contention, and maybe reduce errors? do you think this would help or should we just let it as is (every container has all resources of the machine available to it)?

Reply:
i don't have a good intuition about if it would help with contention, but it seems like a worthy experiment. perhaps 2 CPUs?

my only concerns are about limits for local users (that also use docker compose), and whether this would cause a redpanda process to observe only 1 cpu.

Reply:
coincidentally, I found this while looking for other stuff: https://scylladb.medium.com/the-cost-of-containerization-for-your-scylla-a24559d17d01

Reply:
I think it'd be worth experimenting with those settings and see how it goes. @gousteris wdyt?